pkg/services/ngalert/state/cache.go

```diff
@@ -60,12 +69,12 @@ func (rs *ruleStates) getOrCreate(ctx context.Context, log log.Logger, alertRule
 		}
 	}
 
-	lbs := make(data.Labels, len(extraLabels)+len(ruleLabels)+len(result.Instance))
+	lbs := make(data.Labels, len(extraLabels)+len(labels)+len(result.Instance))
```

Comment:
I think ruleLabels were more specific than "random" labels

Reply:
Should I do the same for annotations and call them `ruleAnnotations`?

Reply:
I was hoping there wasn't a need to be that specific here as there will be no other local variables called labels in this function. However, if I we want to keep using `ruleLabels` then I think I have to the same for annotations and call them `ruleAnnotations`, as otherwise it looks like the annotations don't belong to the rule according to the naming scheme.

Reply:
I think this is a good idea, and it's consistent.

Reply:
Perhaps we use `expandedLabels` and `expandedAnnotations` instead, as these variables are not really the original labels or annotations as they have been expanded?